languageLearning/app/src/main/java/com/example/languageLearning/BridgeActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.example.languageLearning;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class BridgeActivity extends AppCompatActivity {
+
+    Intent i = new Intent(this, ProfExamActivity.class);
```

Comment:
When I try to navigate to this activity, app stops working and shows this line as error.
When I tried putting this line separately to all functions it worked.

Reply:
Thanks for pointing out the problem. I solved the issue as you suggested. Does it work now?

Reply:
Now its working without errors. Thanks